common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -44,7 +44,7 @@ public enum Feature {
         AUTHORIZATION("Authorization Service", Type.DEFAULT),
 
         ACCOUNT_API("Account Management REST API", Type.DEFAULT),
-        ACCOUNT2("Account Management Console version 2", Type.DEFAULT, Feature.ACCOUNT_API),
+        ACCOUNT2("Account Management Console version 2", Type.PREVIEW, Feature.ACCOUNT_API),
```

Comment:
This changes V2 from default to preview.  I think you wanted to change the next line.

Reply:
Ahhh, yeah. Fixed.